CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ Please format the changes as follows:
   + Add configuration for InstrumentationMethod loglevels
 + Updates:
   + Added documentation around the logging infrastructure
++ BugFixes:
+  + Fix codesigning ProfilerProxy
```

Comment:
>Fix [](start = 4, length = 3)

Is this really what you did? Seems like what you did was add code sign verification.

Reply:
Yep, it turned out the typo in the ProfilerProxy project silently failed and did not sign the assembly.

---
In reply to: [356697030](https://github.com/microsoft/CLRInstrumentationEngine/pull/181#discussion_r356697030) [](ancestors = 356697030)